src/UI/Factory.php

```diff
@@ -336,4 +336,63 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Drop Downs reveal a list of interactions that change the system’s status or navigate to
+	 *      a different view.
+	 *   composition: >
+	 *      Drop Down is a clickable, graphically obtrusive control element. It can
+	 *      bear text.
+	 *   effect: >
+	 *      On-click, a list of actions is revealed. Clicking an item will trigger the action indicated.
+	 *      Clicking outside of an opened drop down will close the list of items.
+	 *   rivals:
+	 *      button: >
+	 *          Buttons are used, if single actions should be presented directly in the user interface.
+	 *      links: >
+	 *          Links are used to trigger Interactions that do not change the systems
+	 *          status. They are usually contained inside a Navigational Collection.
+	 *
```

Comment:
And SplitButtons, right? I was wondering in the beginning if a Dropdown is a kind of Button. Still I am not sure about this. But somehow I think these are different things.